style.css

```diff
@@ -0,0 +1,102 @@
+/* import links */
+
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100&family=Roboto:wght@300&display=swap');
+
+/* general */
+
+* {
+  font-family: 'Roboto', sans-serif;
+  box-sizing: border-box;
+}
+
+html {
+  background: #191765;
+}
+
+section {
+  text-align: left;
+}
+
+.multi-story-1,
+.multi-story-2,
+.multi-story-3,
+.multi-story-4,
+h1 {
+  padding: 20px 0 20px 0;
+}
+
```

Comment:
- [ ] It will be better to use flexbox to align your projects. This is a hard requirement for this project. To learn more on how you can use flexbox kindly read this [article](https://www.w3schools.com/css/css3_flexbox.asp)